src/hotspot/share/memory/iterator.hpp

```diff
@@ -41,7 +41,7 @@ class Thread;
 
 // The following classes are C++ `closures` for iterating over objects, roots and spaces
 
-class Closure : public StackObj { };
+class Closure { };
```

Comment:
Why is this required?

Reply:
I just forgot to undo to the original code...